### ISSUE TITLE
Hidden profiles cannot be specified with -p tag from command line.

### ISSUE THREAD

**Issue by jmahlers** [microsoft/terminal#8378]
# Environment

Microsoft Windows [Version 10.0.19042.630]
Windows Terminal
Version: 1.4.3243.0

WSL Ubuntu and others but not relevant.

# Steps to reproduce

Create a profile for example:

```
{
                "guid": "{00000000-35be-5f56-a8ff-afceeeaa6101}",
                "name": "Command-Prompt-In-Place",
                "commandline": "cmd.exe",
                "startingDirectory": ".",
                "hidden": true
}
```
Then, attempt to create an instance of windows terminal from the command line with this profile ie:

`wt.exe -p "Command-Prompt-In-Place"`

# Expected behavior

The specified profile should be chosen. This is useful so that a profile can be triggered by a command that isn't taking up space in the drop down in the app. One example is so that the default profile can start in some useful directory "C:\src" while an otherwise identical profile can start in ".". This is useful because then a profile can start in the current directory without taking up space in the menus. I am using this so I can right click and have "Open windows terminal here" in the context menu. Obviously the profile for "Open windows terminal here" is not needed in the menus.

# Actual behavior
The hidden profile is not targetable and the default profile is chosen.

**Comment by DHowett** [comment]
You're in luck! This was fixed in 1.5 (currently in preview) and is a /duplicate of #4139

**Comment by jmahlers** [comment]
Awesome, sorry I didnt find that before. Thanks!
